src/rdkafka.h

```diff
@@ -204,7 +204,18 @@ typedef enum rd_kafka_timestamp_type_t {
 	RD_KAFKA_TIMESTAMP_LOG_APPEND_TIME  /**< Log append time */
 } rd_kafka_timestamp_type_t;
 
-
+/**
+ * @brief Set external allocator for rd_malloc/rd_calloc/rd_realloc/rd_free.
+ * This has to be set before calling any other rdkafka functions.
```

Comment:
`@warning This function may only be called once and must be called before calling any other rdkafka functions.`